docs/content/develop/properties.md

```diff
@@ -0,0 +1,352 @@
++++
+title = "Define properties"
+description = ""
+weight = 45
+draft = false
+bref = ""
+toc = true
+menu = "develop"
+tags = [ "develop" ]
++++
+
+Property definitions provide a standard, streamlined way to declare the properties and attributes your component needs.  Property definitions provide the following features:
+
+ - Two-way binding between property and attribute values.
+ - Default value initialization.
+ - Data types (for properties).
+ - Observer function to be run when a value changes.
+ - Optional namespacing prefix for attribute names.
+ - Aliasing properties to one another, for phased deprecation.
+ - And more!
+
+## Introduction
+
+To introduce the topic, consider a component that counts how many times something has happened.We can give it the tag `<pfe-count>` and a `count` attribute (`<pfe-count count="0">`) to set the initial count.  We might also want a local property `this.count` for convenient referencing and updating of the count (ex: `this.count += 1` vs `this.setAttribute(this.getAttribute("count") + 1)`).
```

Comment:
```suggestion
To introduce the topic, consider a component that counts how many times something has happened. It may have a tag of `<pfe-count>`, and a `count` attribute (`<pfe-count count="0">`) which sets the initial count.  We might also want a local property `this.count` for convenient referencing and updating of the count (ex: `this.count += 1` vs `this.setAttribute(this.getAttribute("count") + 1)`).
```

Reply:
Sounds good!